packages/apputils/src/mainareawidget.ts

```diff
@@ -314,6 +289,11 @@ export class MainAreaWidget<T extends Widget = Widget>
 
   private _isRevealed = false;
   private _revealed: Promise<void>;
+  private _evtMouseDown = () => {
```

Comment:
Initially I use a _protected_ callback in case someone needs to change the behavior. Do you think we should make it private?